api/src/middlewares/services/SubmitService/SubmitService.ts

```diff
@@ -28,10 +28,11 @@ export class SubmitService {
     const formFields = flattenQuestions(questions);
     const answers = answersHashMap(formFields);
     const reference = metadata?.pay?.reference ?? this.generateId();
+    const type = metadata?.type ?? "affirmation";
 
-    const staffJobId = await this.staffEmailService.send(formFields, "affirmation", { reference, payment: metadata.pay });
-    const userNotifyJobId = await this.notifyEmailService.sendEmailToUser(answers, { reference, payment: metadata.pay });
-    const postNotifyJobId = await this.notifyEmailService.sendEmailToPost(answers);
+    const staffJobId = await this.staffEmailService.send(formFields, "submission", { reference, payment: metadata.pay, type });
```

Comment:
we can add another ticket, or you can fix here. I think it only really matters if staffEmailService.send throws an application error which has the QUEUE_ERROR code